### DISCUSSION TITLE
Copilot does not authenticate while being deactivated.

### DISCUSSION THREAD

**Discussion by MulleDK19** [community/community#40316] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I just bought Copilot and been spending the past hour getting it to work.

All the options in the right click menu were disabled, and it wouldn't ask for authorization. After trying to re-install and delete caches, I finally went into the Copilot settings in Visual Studio and noticed "Enable globally" was set to False. As soon as I set it to True, the authentication message finally popped up.

I don't know how it got disabled, probably from the technical test.

Copilot should probably ask to authorize even if disabled.

**Comment by Akash1134** [comment]
Hi @MulleDK19  Sorry that you're facing issues setting up Copilot. It seems that Copilot is not authenticating your account. The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them and they will help you setup the product :)
